### DISCUSSION TITLE
How to show office number in people search result

### DISCUSSION THREAD

**Discussion by vizsnyai** [microsoft-search/pnp-modern-search#4133] [Q&A] [answered]
How can I display the office value in the people search results list?
The data is displayed on the hover card, but I would like to display it on the extra large card in the results list.

Thanks

**Comment by vishalgoyal2472** [comment]
Do you want to Search for Office location then you can try **OfficeNumber** property.

**Reply by vizsnyai** [reply]
Hi,

No, I want to display the room number. In AD, this is the office attribute.

**Reply by kasperbolarsen** [reply]
People search is still based on the SharePoint User Profile Application as the source, so you will have to look at the various attributes in the User Profile.

The mananged Property OfficeNumber is mapped to the crawled property People:Office which is the room number.

The Managed Property BaseOfficeLocation is mapped to the crawled property People:SPS-Location which is tied to the Term Set "location"

**Reply by vizsnyai** [reply]
Unfortunately, I can't seem to get this information onto the People card. When I open the people card in the results list, the room number is there, but I would like it to appear immediately for the 4-line value.

**Reply by kasperbolarsen** [reply, marked as answer]
You will have to add OfficeNumber to Selected Properties in the Results web part, then add it as a Slot , when edit the Tertiary people field to something like this:
`{{getUserEmail (slot item @root.slots.UserEmail)}}<br>
OfficeNumber:{{slot item @root.slots.OfficeNumber}}`

That should give you something like this
![image](https://github.com/user-attachments/assets/ee0e3c5f-6cb0-4aaa-a482-20f2c9d29e1f)

**Reply by vizsnyai** [reply]
Thanks, it works. :)
